.github/workflows/ci.yml

```diff
@@ -38,27 +38,19 @@ jobs:
         ports:
           - 9200:9200
   
-    strategy:
-      matrix:
-        opensearch-version: [1.2]
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
+
+      - name: Apt update
+        run: sudo apt-get update -y
 
       - name: Apt install
         run: cat Aptfile | sudo xargs apt-get install
 
-      - name: Set up Python
-        uses: actions/setup-python@v2
-        with:
-          python-version: "3.9.1"
-
-      - id: cache
-        uses: actions/cache@v4
+      - name: Install Python 3.9.23
```

Comment:
This could still say `name: Set up Python` as that has been a standard for all our apps. The action details in the CI would mention the version anyway.